docs/StopAreas.md

```diff
@@ -0,0 +1,24 @@
+# Stop Areas
+
+Public transit stations and stops can be modeled using a relation tagged with `public_transport=stop_area`.
```

Comment:
Should this be part of the docs menu? If so, then please add it to `mkdocs.yml`.

Reply:
Done.